internal/tiltfile/tiltfile_test.go

```diff
@@ -1320,28 +1260,24 @@ func TestFilterYamlByApiVersion(t *testing.T) {
 		testyaml.DoggosDeploymentYaml, testyaml.DoggosServiceYaml,
 		testyaml.SnackYaml, testyaml.SanchoYAML))
 	f.file("Tiltfile", `
-k8s_resource_assembly_version(1)
 doggos, rest = filter_yaml('k8s.yaml', name='doggos', api_version='apps/v1')
-k8s_resource('doggos', yaml=doggos)
-k8s_resource('rest', yaml=rest)
+k8s_yaml(doggos)
 `)
-	f.loadResourceAssemblyV1()
 
+	f.load()
 	f.assertNextManifest("doggos", deployment("doggos"))
-	f.assertNextManifest("rest", service("doggos"), deployment("snack"), deployment("sancho"))
+	f.assertNoMoreManifests()
 }
 
 func TestFilterYamlNoMatch(t *testing.T) {
 	f := newFixture(t)
 	defer f.TearDown()
 	f.file("k8s.yaml", yaml.ConcatYAML(testyaml.DoggosDeploymentYaml, testyaml.DoggosServiceYaml))
 	f.file("Tiltfile", `
-k8s_resource_assembly_version(1)
 doggos, rest = filter_yaml('k8s.yaml', namespace='dne', kind='deployment')
-k8s_resource('doggos', yaml=doggos)
-k8s_resource('rest', yaml=rest)
+k8s_yaml(doggos)
 `)
-	f.loadErrString("could not associate any k8s YAML with this resource")
+	f.loadErrString("Empty or Invalid YAML Resource Detected")
```

Comment:
I know you didn't write this error message as part of this PR, but if you feel like changing it, this isn't quite right, should be something like "empty/invalid k8s YAML detected".

Reply:
ya, lemme do that on a separate PR